Makefile

```diff
@@ -11,12 +12,17 @@ help:
 	@echo " make release - build juju-verify and give hints to release it"
 	@echo " make clean - remove unneeded files"
 	@echo " make snap - build juju-verify as a snap"
+	@echo " make documentation - create documentation"
```

Comment:
What about `make docs`? It would much the tox env name, too.

Reply:
That's doesn't work. 
```bash
$ make docs
make: 'docs' is up to date.
```

Reply:
aaah, right. This is because "docs" is a directory. Then +1 on leaving this as it is :+1:

Reply:
It could be added to the `.PHONY` target list, but it probably does not matter either way as we'll use `tox` to run these things, not `make` directly.